overlord/snapstate/handlers.go

```diff
@@ -460,6 +468,52 @@ func (m *SnapManager) cleanupCopySnapData(t *state.Task, _ *tomb.Tomb) error {
 	return nil
 }
 
+func (m *SnapManager) createSnapContext(st *state.State, snapName string) error {
+	contextID := strutil.MakeRandomString(40)
```

Comment:
Hah.. this isn't base64, and it's not 44 chars either, right? Is integration testing still missing? Let's please have it before we commit this.